packages/cxl-ui/scss/cxl-featured-image.scss

```
@@ -0,0 +1,25 @@
+:host {
+  position: relative;
+  display: block;
+  min-width: 360px;
```

```suggestion
  min-width: 320px;
```

At the same time I'm wondering whether it even needs `min-width`. It's block element which means it's 100% wide by default. And on screens smaller than 360 it causes horizontal scrollbar.

---

Updated. Since the element technically doesn't have any content(only a background), without setting a minimum width, it collapses to 0.